Assets/ShooterProject/Scripts/General/GameObjectsPool.cs

```diff
@@ -0,0 +1,116 @@
+using System.Collections.Generic;
+using UnityEngine;
+namespace ShooterProject.Scripts.General
+{
+	public class GameObjectsPool
+	{
+		#region Fields
+
+		private List<GameObject> _objects;
+
+		private bool _autoExpand;
+		private bool _activeByDefault;
+		private Transform _container;
+		private GameObject _objectPrefab;
+		private bool _initialized;
+
+		#endregion
+
+		#region Properties
+
+		public int MaxSize { get; }
+
+		#endregion
+
+		#region Constructors
```

Comment:
Вставь пустую строку
```suggestion
		#region Constructors
		
```